testsuite/integration/microprofile/src/test/resources/org/wildfly/test/integration/microprofile/openapi/service/multimodule/application.xml

```diff
@@ -0,0 +1,15 @@
+<application xmlns="http://java.sun.com/xml/ns/javaee"
+             xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+             xsi:schemaLocation="http://java.sun.com/xml/ns/javaee http://java.sun.com/xml/ns/javaee/application_6.xsd"
+             version="6">
```

Comment:
Please update this to use the Jakarta EE8 schema version,  e.g.
```xml
<application xmlns="http://xmlns.jcp.org/xml/ns/javaee" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://xmlns.jcp.org/xml/ns/javaee http://xmlns.jcp.org/xml/ns/javaee/application_8.xsd" version="8">...</application>

Reply:
done